pkg/streamingpromql/optimize/plan/commonsubexpressionelimination/optimization_pass.go

```diff
@@ -68,6 +70,9 @@ func (e *OptimizationPass) Apply(_ context.Context, plan *planning.QueryPlan) (*
 	e.selectorsInspected.Add(float64(len(paths)))
 	e.selectorsEliminated.Add(float64(selectorsEliminated))
 
+	spanLog := spanlogger.FromContext(ctx, log.NewNopLogger())
+	spanLog.DebugLog("msg", "attempted common subexpression elimination", "inspected_selectors", len(paths), "eliminated_selectors", selectorsEliminated)
```

Comment:
[nit] Let's keep the log labels consistent with the metric names:

```suggestion
	spanLog.DebugLog("msg", "attempted common subexpression elimination", "selectors_inspected", len(paths), "selectors_eliminated", selectorsEliminated)
```